redash/authentication/__init__.py

```diff
@@ -244,7 +244,6 @@ def init_app(app):
     login_manager.init_app(app)
     login_manager.anonymous_user = models.AnonymousUser
 
-    app.secret_key = settings.COOKIE_SECRET
```

Comment:
This is just a cleanup change that was missing in #2970.

Reply:
See other comment about this ⬇️